utils/sflow_test.go

```diff
@@ -0,0 +1,26 @@
+package utils
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestDecodeFlowExpandedSFlow(t *testing.T) {
+	msg := BaseMessage{
+		Src:     []byte{},
+		Port:    1,
+		Payload: getExpandedSFlowDecode(),
+	}
+
+	s := &StateSFlow{
+		Transport: &DefaultLogTransport{},
+		Logger:    log.StandardLogger(),
```

Comment:
Nitpicking: I'd rather not have the import of a library for logging in tests.

Reply:
I'm not very familiar with go but seems like I could just remove the line and it works